monitoring/ceph-mixin/tests_dashboards/util.py

```diff
@@ -85,7 +85,7 @@ def add_default_dashboards_variables(data: Dict[str, Any]) -> None:
 def replace_grafana_expr_variables(expr: str, variable: str, value: Any) -> str:
     """ Replace grafana variables in expression with a value
 
-    It should match the whole word, 'osd' musn't match with the 'osd' prefix in 'osd_hosts'
+    It should match the whole word, 'osd' must not match with the 'osd' prefix in 'osd_hosts'
```

Comment:
Expanding the contraction was explicitly requested https://github.com/ceph/ceph/pull/46824?notification_referrer_id=NT_kwDOACBWLLIzODY3MDU2MDc2OjIxMTkyMTI#discussion_r905779516